### DISCUSSION TITLE
adding SWAYNC and replacing dunst for notications

### DISCUSSION THREAD

**Discussion by JaKooLit** [JaKooLit/Hyprland-Dots#104] [General]
@SherLock707

I dont know if you will be tagged.

So I tried changing over to swaync

but seems dunst still taking over...

not sure why

did you have to uninstall dunst?

**Comment by SherLock707** [comment]
I was running dunst launched from your script, I simply did `killall dunst` then launched sway using `sway &`

**Comment by JaKooLit** [comment]
yeah did that as well.. but after reboot, dunst still taking over.....

hmmmm

**Reply by SherLock707** [reply]
Oh, even if you remove dunst from hyp Execs conf?

**Reply by JaKooLit** [reply]
yes sorry.. of course I remove that exec-once dunst

ultimately I have to remove dunst

**Reply by SherLock707** [reply]
I'm running Swaync as daily driver without uninstalling Dunst from your dots..

**Reply by SherLock707** [reply]
I only tried this on Arch..

**Reply by JaKooLit** [reply]
yeah I am also testing in Arch.. since its alot faster to play around here 🤣


anyway, Ill figure it out

**Reply by SherLock707** [reply]
I has dunst, mako, deadd notification center and swaync in my system same time, swaync ran without issue..
(Right now have only dunst and swaync installed)

**Reply by JaKooLit** [reply]
Its ok.. Ill figure it out.. just something on my mind before implementing as I need to adjust to all my distro hyprland scripts 😀

**Reply by SherLock707** [reply]
May be refresh script is getting trigged before execs conf somehow?


Oh, forgot to add, I'm using your [v2.2](https://github.com/JaKooLit/Hyprland-Dots/releases/tag/v2.2) dots (Since I made few changes on top of it, mostly waybar looks, didn't merge your latest release yet.)

**Reply by JaKooLit** [reply]
no no its fine... 


Ill make necessary adjustment...

lots of work to do...

**Reply by SherLock707** [reply]
Love your dot, thanks for your handwork man, let me know if I can do anything to help.

my current setup thanks to you.
![image](https://github.com/JaKooLit/Hyprland-Dots/assets/26952545/b0f66004-5f50-483e-8308-1ca01d2cc49c)

**Comment by JaKooLit** [comment]
I just pushed some changes... if you have time please try out...

background is not changing :( for dark and light function.. annoying

**Reply by SherLock707** [reply]
I'll be AFK for 2-3 days, I'll still try to check these out sooner. Thanks.

I only used dark mode so far, I'll check if I can get it to work.

**Comment by SherLock707** [comment]
I tried updating following:

![image](https://github.com/JaKooLit/Hyprland-Dots/assets/26952545/e99445f2-aa6e-4985-87a2-4f333564d384)

![image](https://github.com/JaKooLit/Hyprland-Dots/assets/26952545/6c216a54-bc3e-4656-b1ac-887d12328926)

![image](https://github.com/JaKooLit/Hyprland-Dots/assets/26952545/f6d73c23-3e67-46d2-893b-cb5bcbb1a978)

![image](https://github.com/JaKooLit/Hyprland-Dots/assets/26952545/8abf3f7b-d74a-4e8e-9889-098997230eb8)


I tried white too, seems to be setting.
updating this to white and dark should work I feel. The defined background colour on top is not used I guess.

**Reply by SherLock707** [reply]
There are more configs here we can refer: [https://github.com/ErikReider/SwayNotificationCenter/discussions/183](https://github.com/ErikReider/SwayNotificationCenter/discussions/183)

**Comment by JaKooLit** [comment]
Its ok bro. Thanks.. I am just finalizing my tweak
